.expeditor/scripts/verify/build_mac_package.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash
+#!/usr/local/bin/bash
```

Comment:
What are the pros and cons of this versus `#!/usr/bin/env bash`?

Reply:
The pro is that I can be reasonably certain `/usr/local/bin/bash` is the Homebrew installed bash.  `/usr/bin/env` bash would probably also resolve to that, but would be dependent on how the PATH was constructed.    

I personally prefer the failure case of `/usr/local/bin/bash` to  `/usr/bin/env bash` resolving to the wrong bash; `/usr/local/bin/bash command not found` vs `declare -g invalid option` (if you got system bash),  if something were to go sideways on us.